### PR TITLE
Channels support

### PR DESCRIPTION
**Pull request by jfdenise**
* Updated tools to build images and run tests with the local maven repo
* Removed all feature-packs versions from test applications.
* Updates test and features applications to rely on org.jboss.eap.channels:eap-8.0-beta (tha tcontains eap-datasources and eap-cloud feature-packs

### CODE REVIEW

**Comment by jfdenise** [comment]
@jmesnil , here we are ;-). All tests are run with channels.

File: test/vanilla-eap/test-app-local-provisioning/galleon/provisioning.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 
 <installation xmlns="urn:jboss:galleon:provisioning:3.0">
-    <feature-pack location="org.jboss.eap:wildfly-ee-galleon-pack:8.0.0.Beta-redhat-20220408">
+    <feature-pack location="org.jboss.eap:wildfly-ee-galleon-pack:">
```
Comment (jmesnil) [inline test/vanilla-eap/test-app-local-provisioning/galleon/provisioning.xml:4]: is that correct or should the trailing `:` be removed?

File: tools/build-custom-image.sh [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+#Ultra custom image with custom cloud-fp, custom eap maven-plugin, custom wildfly maven plugin.
```
Comment (jmesnil) [inline tools/build-custom-image.sh:2]: when would we need both eap maven plugin *and* wildfly maven plugin?

File: tools/ocp-settings.xml [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+      </pluginRepositories>
+    </profile>
+
+    <!-- JBoss EAP Maven repository (HTTP version, disabled by default) -->
+    <profile>
+      <id>jboss-eap-repository-insecure</id>
+      <repositories>
+        <repository>
+          <id>redhat-ga-repository</id>
+          <url>http://maven.repository.redhat.com/ga/</url>
```
Comment (jmesnil) [inline tools/ocp-settings.xml:103]: meh, is there any good reason to continue to use http URLs?

File: test/vanilla-eap/test-app-local-provisioning/galleon/provisioning.xml (hunk 1 above)
Comment (jfdenise) [reply test/vanilla-eap/test-app-local-provisioning/galleon/provisioning.xml:4]: Yes, we are not handling the provisioning.xml content to deal wit hthe trailing ":", it is needed.

File: tools/build-custom-image.sh (hunk 2 above)
Comment (jfdenise) [reply tools/build-custom-image.sh:2]: That is when new features appear in wildfly plugin prior any release. This script is really for ultra custom cases during WIP.

File: tools/ocp-settings.xml (hunk 3 above)
Comment (jfdenise) [reply tools/ocp-settings.xml:103]: This is a copy of the settings used in the image. The un-secure (http) profiles are there and disabled by default.

**Review by jmesnil — APPROVED** [review]

**Pull request merged**